features/roomdetails/impl/src/test/kotlin/io/element/android/features/roomdetails/rolesandpermissions/permissions/ChangeRoomPermissionsPresenterTests.kt

```diff
@@ -179,6 +191,8 @@ class ChangeRoomPermissionsPresenterTests {
                 assertThat(currentPermissions?.roomName).isEqualTo(MODERATOR.powerLevel)
                 assertThat(saveAction).isEqualTo(AsyncAction.Success(Unit))
             }
+            assertThat(analyticsService.capturedEvents).hasSize(8)
+            assertThat(analyticsService.capturedEvents.all { it is RoomModeration }).isTrue()
```

Comment:
The typo may have been spotted if the tests were more deterministic. Actually, I checked again the code when I saw the test 😛

Reply:
Yeah, I'll change it so it checks for every single change.